workloads/kube-burner/README.md

```diff
@@ -0,0 +1,35 @@
+# Kube-burner e2e benchmarks
+
+The purpose of these scripts is to run a kube-burner workload steered by ripsaw. There are 3 types of workloads at the moment:
+
+- cluster-density
+- kubelet-density
+- kubelet-density-heavy
+
+## Environment variables
+
+All scripts can be tweaked with the following environment variables:
+
+| Variable         | Description                         | Default |
+|------------------|-------------------------------------|---------|
+| **QPS**              | Queries/sec                     | 10      |
+| **Burst**            | Burst queries                   | 10      |
+| **ES_SERVER**        | Elastic search endpoint         | https://search-cloud-perf-lqrf3jjtaqo7727m7ynd2xyt4y.us-west-2.es.amazonaws.com|
+| **ES_PORT**          | Elastic search port             | 443|
+| **ES_INDEX**         | Elastic search index            | ripsaw-kube-burner|
+| **PROM_URL**         | Elastic search endpoint         | https://prometheus-k8s.openshift-monitoring.svc.cluster.local:9091|
+| **JOB_TIMEOUT**      | kube-burner's job timeout       | 7200|
+| **WORKLOAD_NODE**    | Workload node name              | "" (don't pin kube-burner to any server)|
+| **TAINT_NODE**       | Taint the workload node         | 0 (don't taint)|
+| **CERBERUS_URL**     | URL to check the health of the cluster using [Cerberus](https://github.com/openshift-scale/cerberus) | "" (don't check)|
+
+### cluster-density variables
+
+The `cluster-density` workload supports the environment variable **JOB_ITERATIONS**. This variable configures the number of cluster-density jobs iterations to perform (1 namespace per iteration). By default 1000
```

Comment:
We might want to mention the number of objects which get created in each namespace to be able to estimate the load cluster will be able to handle and set the number of namespaces accordingly. Thoughts?